app/tests/spec/views/coppa/coppa-age-input.js

```diff
@@ -67,6 +67,29 @@ define(function (require, exports, module) {
       });
     });
 
+    describe('onKeyDown', function () {
+      it('force digit input', function () {
+        var age = $('#age');
+        age.val('1.1');
```

Comment:
A better approximation of real life would be to generate three individual `keydown` events, one for each character, similar to lines 84/85. Once the three events are triggered, then call `assert.equal(age.val(), '11');`